index.html

```diff
@@ -0,0 +1,212 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <link rel="preconnect" href="https://fonts.googleapis.com" />
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Josefin+Sans&display=swap"
+      rel="stylesheet"
+    />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <title>Matt's Portfolio</title>
+  </head>
+  <body>
+    <header>
+      <img
+        class="nameLogo"
+        src="images/matt-knight-logo-new.svg"
+        alt="Matt Knight logo"
+      />
+      <nav>
+        <ul class="nav-container">
+          <li class="nav-item"><a href="#aboutmelink">About Me</a></li>
+          <li class="nav-item"><a href="#projectslink">Projects</a></li>
+          <li class="nav-item"><a href="#footer">Contact</a></li>
+          <li class="nav-item"><a href="images/Matt-Knight-Cv-WebDev.pdf">Resume</a></li>
+        </ul>
+      </nav>
+    </header>
+
+    <div class="banner-area"></div>
+
+    <section class="about-me">
+      <img
+        src="images/Matt-knight-photo.jpg"
+        alt="Matt Knight portrait"
+        class="portfolio-portrait"
+      />
+      <div id="aboutmelink">
+        <h2>About Me</h2>
+        <p>
+          I have recently begun my journey as a web developer.
+          Some of the skills I have aquired so far are:
+          <ul>
+              <li>HTML wireframes</li>
+              <li>CSS styling</li>
+              <li>Flex-boxes</li>
+              <li>CSS variables</li>
+              <li>Typography</li>
+              <li>Media queries</li>
+              <li>Pseudo classes</li>
+          </ul>
+          I hope to be learning much more over the next few months and provide you with some examples of my work and projects below!
+          Aside from programming, my hobbies outside of work include science fiction, playing the piano and keeping my neighbours up.
+        </p>
+      </div>
+    </section>
+    <!-- Cards in wrapper below -->
+    <main>
+      <div id="projectslink" class="wrapper">
+        <!-- Card 1 large -->
+        <a href="https://github.com/Mattrix01/semantic-html-refactor">
+            <div class="card-large">
```

Comment:
There's something off with the indentations of lines here - make sure your Prettier is working correctly as it should do this automatically for you on save (unless there are any errors in your document!).